### PR TITLE
Tests(Component): fix panel edit tests

### PR DESCRIPTION
**Pull request by hyahiaoui-ext**
## Description

Fixing random fail for PanelEdit component testing

**Fixes** # MON-24125

## Type of change

- [c] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software

## Target serie

- [ ] 21.10.x
- [ ] 22.04.x
- [ ] 22.10.x
- [ ] 23.04.x
- [x] 23.10.x (master)

#### Community contributors & Centreon team

- [x] I have followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [x] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

File: centreon/www/front_src/src/CloudNotificationsConfiguration/Panel/specs/PanelEdit.cypress.spec.tsx [hunk 1]
```diff
@@ -447,7 +453,7 @@ describe('Edit Panel', () => {
       cy.findByRole('checkbox').should('not.be.checked').and('be.disabled');
     });
 
-    cy.get('#panel-content').scrollTo('bottom');
+    cy.get('#panel-content').scrollTo('center');
```
Comment (kduret) [inline centreon/www/front_src/src/CloudNotificationsConfiguration/Panel/specs/PanelEdit.cypress.spec.tsx:456]: why not to bottom ?
snapshots are totally differents

Comment (hyahiaoui-ext) [reply centreon/www/front_src/src/CloudNotificationsConfiguration/Panel/specs/PanelEdit.cypress.spec.tsx:456]: @kduret When Bottom is applied the area of "notification channels" are not visible in the viewport ^^

Comment (hyahiaoui-ext) [reply centreon/www/front_src/src/CloudNotificationsConfiguration/Panel/specs/PanelEdit.cypress.spec.tsx:456]: Whereas the test case talks about notification channels ^^

File: centreon/www/front_src/src/CloudNotificationsConfiguration/Panel/specs/PanelEdit.cypress.spec.tsx [hunk 2]
```diff
@@ -142,6 +144,8 @@ describe('Edit Panel', () => {
 
     cy.get('#panel-content').scrollTo('top');
 
+    cy.clock();
+
```
Comment (kduret) [inline centreon/www/front_src/src/CloudNotificationsConfiguration/Panel/specs/PanelEdit.cypress.spec.tsx:148]:
```suggestion
```

File: centreon/www/front_src/src/CloudNotificationsConfiguration/Panel/specs/PanelEdit.cypress.spec.tsx [hunk 3]
```diff
@@ -158,6 +162,8 @@ describe('Edit Panel', () => {
...
```
Comment (kduret) [inline centreon/www/front_src/src/CloudNotificationsConfiguration/Panel/specs/PanelEdit.cypress.spec.tsx:166]:
```suggestion
```

File: centreon/www/front_src/src/CloudNotificationsConfiguration/Panel/specs/PanelEdit.cypress.spec.tsx (hunk 1 above)
Comment (kduret) [reply centreon/www/front_src/src/CloudNotificationsConfiguration/Panel/specs/PanelEdit.cypress.spec.tsx:456]: then you should use [scrollIntoView](https://docs.cypress.io/api/commands/scrollIntoView) to scroll to the notification section

Comment (hyahiaoui-ext) [reply centreon/www/front_src/src/CloudNotificationsConfiguration/Panel/specs/PanelEdit.cypress.spec.tsx]: @kduret done !

**Review by Thebarda — APPROVED** [review]
Yeaaah

**Review by kduret — APPROVED** [review]

**Pull request merged**